docs/datagrid/columns.md

```diff
@@ -1,152 +1,108 @@
-Column options in the Datagrid
-===
-The DataGrid provides flexible configuration options to meet your specific needs.
+# Columns options
+
+Many of the available configuration options in DataGrid applies to the columns and their corresponding row and header cells.
 
 ## Defaults
-By default, the options from the [columnDefaults](https://api.highcharts.com/dashboards/#interfaces/DataGrid_Options.ColumnOptions) property are applied to all columns.
-For instance, you can allow editing of cells in all columns in `columnDefaults` instead of applying an option to each column separately.
 
 ```js
-columnDefaults: {
-  cells: {
-    editable: true
-  }
+{
+    columnDefaults: {
+        cells: {
+            editable: true
+        },
+        sorting: {
+            sortable: false
+        }
+    },
+    columns: [
+        {
+            id: "product",
+            sorting: {
+                sortable: true
+            }
+        }
+    ]
 }
 ```
 
-## Column headers
-In columns options, use [header.format](https://api.highcharts.com/dashboards/#interfaces/DataGrid_Options.ColumnHeaderOptions#format) to customize the header content for that column.
+Default options for all columns in the DataGrid, such as the column sorter, column resizer, value editor, cell format etc., can be defined in the `columnDefaults` object, and the `columns[]` array of objects can be used to override defaults in selected columns if needed.
 
-```js
-columns: [{
-  id: 'column1',
-  header: {
-    format: 'Custom header text'
-  }
-}]
-```
+Note that most of the options in `columnDefaults` are mirrored 1:1 in the `columns[]` array of objects.
 
-### Customize headers
-By default, all columns from the DataTable are imported into the DataGrid in the order they are declared.
-This can be modified using the [header](https://api.highcharts.com/dashboards/#interfaces/DataGrid_Options.Options-1#header) option to define columns (or order) that you would like to display.
+## Header
 
 ```js
-header: ['a', 'price', 'weight']
+{
+  columns: [
+    {
+      id: "product",
+      header: {
+        className: "custom_header_class",
+        format: "Fruit ({id})",
+        formatter: function () {
```

Comment:
It might be misleading to have both `format` and `formatter` - I would pick one. We have link to advanced formatting 
The same for `header`

Reply:
I would keep the `format `here because the simplest solution is often the best in a "getting started" context. Perhaps add a text along the line: "for more advanced formatting the `formatter` function can be used". Either with a separate example or a link to the API doc.

Reply:
Yep, we should recommend the `format` as default because it is serializable.